MobileFilter/sections/specific_web.txt

```diff
@@ -43,6 +43,7 @@ anwap.club,anwap.film,anwap.mobi##body > div[class] > a > strong
 !
 m.salihlisektorgazetesi.com##iframe[src^="/mobil-reklam-"]
 ||m.salihlisektorgazetesi.com/mobil-reklam-
+||eysmaa.pw^$domain=m.lawen44.com
```

Comment:
this is ad domain. Block it in adservers.
And add new rules near `://m.*.top/*.js?ssid=_`

Reply:
Moved rule to adservers.